### ISSUE TITLE
[QUESTION] How to modify RRTMG BPCH diagnostics/files frequency?

### ISSUE THREAD

**Issue by FeiYao-Edinburgh** [geoschem/geos-chem#480]
Hi there,

I am using GC.12.9.3. I found [here](http://wiki.seas.harvard.edu/geos-chem/index.php/Coupling_GEOS-Chem_with_RRTMG#In_GEOS-Chem_12.3.0_and_later_versions) that you still recommend using bpch format for RRTMG diagnostics. However, I do not know how to modify the freq for bpch diagnostics and the freq for bpch files. That is is an easy operation for netcdf files by contrast. Currently with the following settings I can only obtain daily bpch diagnostics on daily bpch files. I would like to ask if there are place where I can place some setting like `Frequency [sec]         : 10800` and/or `Duration [sec]         : 864000` to write bpch diagnostics every 3 hours and create bpch files every day?
```
%%% OUTPUT MENU %%%     : 123456789.123456789.123456789.1--1=ZERO+2=BPCH
Schedule output for JAN : 3333333333333333333333333333333
Schedule output for FEB : 33333333333333333333333333333
Schedule output for MAR : 3333333333333333333333333333333
Schedule output for APR : 333333333333333333333333333333
Schedule output for MAY : 3333333333333333333333333333333
Schedule output for JUN : 333333333333333333333333333333
Schedule output for JUL : 3333333333333333333333333333333
Schedule output for AUG : 3333333333333333333333333333333
Schedule output for SEP : 333333333333333333333333333333
Schedule output for OCT : 3333333333333333333333333333333
Schedule output for NOV : 333333333333333333333333333333
Schedule output for DEC : 3333333333333333333333333333333

%%% DIAGNOSTIC MENU %%% :
Binary punch file name  : trac_avg.geosfp_4x5_benchmark.YYYYMMDDhhmm
Diagnostic Entries ---> :  L   Tracers to print out for each diagnostic
ND03: Hg emissions, P/L :  0   all
ND06: Dust aer source   :  1   all
ND44: Drydep flx/vel    :  1   all
ND53: POPs Emissions    :  0   all
ND59: TOMAS aerosol emis:  0   all
ND60: TOMAS rate        :  0   all
ND61: TOMAS 3D rate     :  0   all
ND72: Radiative output  :  1   2 6 9
```

**Comment by msulprizio** [comment]
That is correct. The minimum frequency for the bpch diagnostics is daily mean output as specified by the Output Menu in input.geos. You can save certain output more frequently using the timeseries bpch diagnostics, but the RRTMG diagnostics weren't enabled for those output. 

In GEOS-Chem 13.0.0, the RRTMG diagnostics have been completely enabled in HISTORY.rc for saving out to netCDF. See issue https://github.com/geoschem/geos-chem/issues/350 for more details. We're hoping to do a provisional release of 13.0.0 in the next few weeks, once the 1-month and 1-year benchmarks have been approved by developers and  the GEOS-Chem Steering Committee. You can update to that version when it's available to save out the RRTMG diagnostics at the frequency, duration, and mode of your choosing.
